docs/releases/1.10.txt

```diff
@@ -97,7 +97,13 @@ Minor features
 :mod:`django.contrib.contenttypes`
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-* ...
+* :class:`~django.contrib.contenttypes.fields.GenericForeignKey` no longer
```

Comment:
Okay, bug fixes aren't typically mentioned in major release notes (unless there are significant backwards-incompatible ramifications in which case they might be mentioned in that section).